kim_cole/assignments/Portfolio_assignment/Portfolio.html

```diff
@@ -0,0 +1,87 @@
+        		<img class="languageimages" src="images/angular.png" alt="" width="100" height="150">
+        		<img class="languageimages" src="images/nodejs.png" alt="" width="100" height="150">
+        		<img class="languageimages" src="images/HTML.png" alt="" width="100" height="150">
+        		<img class="languageimages" src="images/css.png" alt="" width="100" height="150">
+        		<img class="languageimages" src="images/socket.png" alt="" width="100" height="150">
+        		</div>
+        		<!-- wrapped a tag inside div to solve css centering issues -->
+        		<div class="button"><a class="pushbutton">Family Contacts</a></div>
+        	</div>
+        	
+        	<div class="columns">
+        		<h3 class="titlebar">fiveEleven</h3>
+        			<img class="columnimage" src="images/fiveEleven.png" alt="" width="100" height="150">
+        			<p class="columntext">Five Eleven is a data visualization project built on the Python software stack. The application transforms the developer job-hunt into a more visual, user friendly experience. Based on location, technology popularity, and other vast data sets, users may easily visualize lthe varying sizes and concentrations of the junior developer job market across the nation.</p>
+        		<h4 class="Technologies">Technologies:</h4>
+        		<div><img class="languageimages" src="images/python.png" alt="" width="100" height="150">
+        		<img class="languageimages" src="images/django.png" alt="" width="100" height="150">
+        		<img class="languageimages" src="images/d3.png" alt="" width="100" height="150">
+        		<img class="languageimages" src="images/ajax.png" alt="" width="100" height="150">
+        		<img class="languageimages" src="images/sqlite.png" alt="" width="100" height="150">
+        		<img class="languageimages" src="images/HTML.png" alt="" width="100" height="150">
+        		<img class="languageimages" src="images/css.png" alt="" width="100" height="150">
+        		<img class="languageimages" src="images/JavaScript.png" alt="" width="100" height="150">
+        		<img class="languageimages" src="images/materialize.png" alt="" width="100" height="150">
+        		</div>
+        		<!-- wrapped a tag inside div to solve css centering issues -->
+        		<div class="button"><a class="pushbutton">fiveEleven</a></div>
+        	</div>
+        	</div>
+   		
+         <div id="Bottom">
+         	<p id="email">jdoe@gmails.com | 206-555-1212</p>
+         		<ul>
+         		<li><img class="footerlogos" src="images/github.png" alt="" width="100" height="150">jaynedev</li>
+         		<li><img class="footerlogos" src="images/linkedin.png" alt="" width="100" height="150">jayne-doe</li>
+         		<li><img class="footerlogos" src="images/twitter.png" alt="" width="100" height="150">jdtweets</li>
+         		<li><img class="footerlogos" src="images/codepen.png" alt="" width="100" height="150">jaynedev</li>
+         		</ul>
+         </div>	
+
+   </body>
```

Comment:
couple things I noticed in the html that you may want to focus on.  Indentation is a bit off and try not to create divs that only have one element inside of it.  Also, instead of giving each image a class, another way to do it that would create less typing would be to just select it more specifically through the parent-child relationship...for example, instead of all the footer images each having a class, you could also select them by #bottom img { css styling here }.  That way you're only styling the images in the bottom div.